.github/workflows/coverity.yml

```diff
@@ -14,7 +14,7 @@ permissions:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Nice! Now that all the dependencies are new enough both Coverity and CodeQL cover all of them finally.